packages/teams-js/CHANGELOG.md

```diff
@@ -12,32 +12,9 @@ Fri, 13 May 2022 22:32:13 GMT
 
 - Promote 2.0.0 beta changes to stable. TeamsJS can be used to write applications with support in multiple Microsoft 365 hosts, including Teams, Outlook, and Office.
```

Comment:
Should we remove this if we are structuring the change log across all beta releases?

Reply:
I'm going to keep it in.